apps/dashboard/src/main/java/com/akto/listener/RuntimeListener.java

```diff
@@ -1,52 +1,108 @@
+        Context.accountId.set(1_000_000);
+        Main.initializeRuntime();
+        httpCallParser = new HttpCallParser("userIdentifier", 1, 1, 1, false);
+        aktoPolicy = new AktoPolicy(RuntimeListener.httpCallParser.apiCatalogSync, false);
 
+        try {
+            initialiseDemoCollections();
+        } catch (Exception e) {
+            loggerMaker.errorAndAddToDb("Error while initialising demo collections: " + e, LoggerMaker.LogDb.DASHBOARD);
+        }
     }
-    
+
+    public void initialiseDemoCollections() {
+        AccountSettings accountSettings = AccountSettingsDao.instance.findOne(new BasicDBObject());
+        if (accountSettings != null && accountSettings.getDemoCollectionCreateTime() > 0) {
+            return;
+        }
+
+        // get har file from github
+        String url = "https://raw.githubusercontent.com/avneesh-akto/tests-library/temp/har_file/juice_onboarding_aws.har";
+        String harString = "";
+        try {
+            harString = new Scanner(new URL(url).openStream(), "UTF-8").useDelimiter("\\A").next();
+        } catch (IOException e) {
+            loggerMaker.errorAndAddToDb("Error downlaoding from github: " + e, LoggerMaker.LogDb.DASHBOARD);
+            return;
+        }
+
+        String tokensUrl = "https://raw.githubusercontent.com/avneesh-akto/tests-library/temp/har_file/juice_shop_tokens";
+        Map<String, String> tokens = new HashMap<>();
+        try {
+            String tokenJsonString = new Scanner(new URL(tokensUrl).openStream(), "UTF-8").useDelimiter("\\A").next();
+            tokens = new Gson().fromJson(tokenJsonString, Map.class);
+        } catch (IOException e) {
+            loggerMaker.errorAndAddToDb("Error downloading from github: " + e, LoggerMaker.LogDb.DASHBOARD);
+            return;
+        }
+
+        String victimToken = tokens.get("victimToken");
+        harString = harString.replaceAll("\\{\\{AKTO\\.token\\}\\}", victimToken);
+
+        // process har file
+        HarAction harAction = new HarAction();
+        harAction.setHarString(harString);
+        harAction.setApiCollectionName("juice_shop_demo");
+        Map<String, Object> session = new HashMap<>();
+        harAction.setSession(session);
+        // todo: skipKafka = true for onPrem also
+        try {
+            harAction.execute();
+        } catch (IOException e) {
+            loggerMaker.errorAndAddToDb("Error: " + e, LoggerMaker.LogDb.DASHBOARD);
+        }
+
+        // auth mechanism
+        String attackerKey = tokens.get("attackerKey");
+        String attackerToken  = tokens.get("attackerToken");
+        List<AuthParam> authParamList = new ArrayList<>();
+        authParamList.add(new HardcodedAuthParam(AuthParam.Location.HEADER, attackerKey, attackerToken, true));
+        AuthMechanism authMechanism = new AuthMechanism(
+             authParamList, new ArrayList<>(), "HARDCODED"
+        );
+        AuthMechanismsDao.instance.insertOne(authMechanism);
```

Comment:
Once the demo collection's testing is executed, we will also need to delete these tokens, right?